### PR TITLE
Feature/swap message with image

### PR DESCRIPTION
**Pull request by tiagojpdias**
- If message is editable then swap its content with the image
- Delete all (but the edited one) messages triggering the bot and warn users via DM

### CODE REVIEW

File: src/events/message.js [hunk 1]
```diff
@@ -6,9 +6,9 @@ const { getTags } = require('../boot');
 const retries = new Map();
 
 function handleRetry(author, retryThreshold = config('bot.msgRetryThreshold')) {
-  const retryCount = retries.get(author.username) + 1;
+  const retryCount = retries.get(author.id) + 1;
```
Comment (quetzyg) [inline src/events/message.js:9]: Yup, better this way. :+1:

File: src/events/message.js [hunk 2]
```diff
@@ -34,67 +34,101 @@ function throttleUser(
...
+      try {
+        const warned = await author.send(
+          '**EI _BORDA_, UMA TAG DE CADA VEZ, OK?!?!**',
+          { files: [attachment] },
+        );
 
-      return;
-    }
+        if (warned) {
+          await author.send(`Tags :: ${allowedTags.join(' , ')}`);
```
Comment (quetzyg) [inline src/events/message.js:77]: What about `', '` instead? Or does it look worse when displaying?

File: src/events/ready.js [hunk 3]
```diff
@@ -1,34 +1,18 @@
...
-    await user.setStatus('dnd');
-    await user.setActivity(config('activities.exit'));
-
-    const goodbyeImage = new Attachment(getReservedImage('goodbye'));
-    const messageSent = await client.channels
-      .get(config('bot.mainChannelId'))
-      .send(config('messages.exit'), {
-        files: [goodbyeImage],
-      });
+    const newStatus = await user.setStatus('dnd');
```
Comment (quetzyg) [inline src/events/ready.js:12]: Does the Discord library have constants for these values? It would read better.

**Review by quetzyg — APPROVED** [review]
Looks good, just left a few comments.

File: src/events/message.js (hunk 2 above)
Comment (tiagojpdias) [reply src/events/message.js:77]: Don't know... Did it this way to make less error prone when the user copy/paste it.

@musemartins mind to untie this?

File: src/events/ready.js (hunk 3 above)
Comment (tiagojpdias) [reply src/events/ready.js:12]: Will check.

Comment (tiagojpdias) [reply src/events/ready.js:12]: I haven't found a way.

File: src/events/message.js (hunk 2 above)
Comment (tiagojpdias) [reply src/events/message.js:77]: Will leave it as it is.

File: src/events/ready.js (hunk 3 above)
Comment (quetzyg) [reply src/events/ready.js:12]: Fuck it, then.

**Pull request merged**